Jenkinsfile.conda

```diff
@@ -1,98 +1,72 @@
 pipeline {
-    agent{
-        docker {
-            image 'lsstts/conda_package_builder:latest'
-            alwaysPull 'true'
-            args "--env TS_CONFIG_OCS_DIR=/home/saluser/ts_config_ocs --env LSST_DDS_DOMAIN=citest --env PATH=${PATH}:/home/saluser/develop/ts_mteec/bin  --env PYTHONPATH=${PYTHONPATH}:/home/saluser/develop/ts_mteec/python -u root --entrypoint=''"
-        }
-    }
+    agent any
     environment {
-        package_name = "ts-mteec"
-        OSPL_HOME="/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux"
-    }
-    options {
-        buildDiscarder(
-            logRotator(
-                artifactDaysToKeepStr: '',
-                artifactNumToKeepStr: '',
-                daysToKeepStr: '14',
-                numToKeepStr: '10'
-                ))
-        disableConcurrentBuilds()
+        package_name = "mteec"
+        package_version = sh(returnStdout: true, script: "git describe --tags --always --dirty").trim()
+        dockerImageName = "lsstts/conda_package_builder:latest"
+        container_name = "salobj_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
     }
+
     stages {
-        stage("Clone ts_config_ocs"){
+        stage("Pull Docker Image") {
             steps {
-                withEnv(["HOME=${env.WORKSPACE}"]) {
-                    sh """
-                    cd /home/saluser
-                    git clone https://github.com/lsst-ts/ts_config_ocs.git
-                    """
+                script {
+                sh """
+                docker pull ${dockerImageName}
+                """
                 }
             }
         }
-        stage("Create Conda package") {
-            when {
-                buildingTag()
-            }
+        stage("Start builder"){
             steps {
-                withEnv(["HOME=${env.WORKSPACE}"]) {
+                script {
                     sh """
-                    cd ${HOME}/conda
-                    source /home/saluser/miniconda3/bin/activate
-                    conda config --add channels conda-forge
-                    conda config --add channels lsstts
-                    source ${OSPL_HOME}/release.com
-                    conda build --prefix-length 100 .
+                    docker run --name ${container_name} -di --rm \
```

Comment:
Better replace this with docker integration? I.e. def docker = docker.image($dockerImageName) and use withEnv to supply the environment? See ts_m1m3support/Jenkinsfile for an example